app/models/concerns/hostext/operating_system.rb

```diff
@@ -0,0 +1,55 @@
+module Hostext
+  module OperatingSystem
+    extend ActiveSupport::Concern
```

Comment:
I agree, but it was a cheap and simple way to fix my problem with https://github.com/theforeman/foreman_discovery/pull/305 without making me write any (possibly wrong logic). I'll investigate how to make that class.